ftdetect/just.vim

```diff
@@ -1,5 +1 @@
-au VimEnter,BufWinEnter,BufRead,BufNewFile *.just setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile Justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile .justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile .Justfile setlocal filetype=just
+au VimEnter,BufWinEnter,BufRead,BufNewFile {.,}[Jj]ustfile,*.just setlocal filetype=just | setlocal commentstring=#\ %s
```

Comment:
```suggestion
au VimEnter,BufWinEnter,BufRead,BufNewFile {.,}justfile\c,*.just setlocal filetype=just | setlocal commentstring=#\ %s
```

It seems like `\c` does work

Reply:
Nice catch, apologies for not following up. Somehow these comments didn't land in my inbox

Reply:
No worries, thanks for the patch 👍